packages/testkit-backend/src/request-handlers.js

```diff
@@ -153,7 +153,14 @@ export function SessionRun (context, data, wire) {
   Promise.all(observers.map(obs => obs.completitionPromise()))
     .catch(_ => null)
     .then(_ => {
-      const result = session.run(cypher, params, { metadata, timeout })
+      let result
+      try {
+        result = session.run(cypher, params, { metadata, timeout })
+      } catch (e) {
+        console.log('got some err: ' + JSON.stringify(e))
+        wire.writeError(e)
+        return
+      }
```

Comment:
Is it a leftover? The backend should be able to capture this kind of errors in the upstream. If it is not, there is a bug in the backend. 

Same for the other block like this.

Reply:
Actually It is correct, my bad 🛏️